server/index.js

```diff
@@ -47,7 +47,7 @@ async function buildDocumentFromURL(url) {
     // When you're running the dev server and build documents
     // every time a URL is requested, you won't have had the chance to do
     // the phase that happens when you do a regular `yarn build`.
-    document.translations = findDocumentTranslations(document);
+    document.translations = findDocumentTranslationsByURL(document);
```

Comment:
nit: I think `findDocumentTranslationsByURL` accepting a `document` is a little bit misleading maybe we wanna pass just `document.url`

Reply:
You're right. I'm glad you stopped me. Fixed now.